intern/openmesheffect/blender/intern/mfxCallbacks.cpp

```diff
@@ -211,23 +469,58 @@ OfxStatus before_mesh_release(OfxHost *host, OfxMeshHandle ofx_mesh) {
 
   printf("Converting ofx mesh into blender mesh...\n");
 
-  // Points (= Blender's vertex)
-  for (int i = 0 ; i < point_count ; ++i) {
-    copy_v3_v3(blender_mesh->mvert[i].co, point_data + (i * 3));
+  // copy OFX points (= Blender's vertex)
+  for (int i = 0 ; i < ofx_point_count; ++i) {
+    copy_v3_v3(blender_mesh->mvert[i].co, (float*)((char*)point_data + i*point_stride));
```

Comment:
Surprising how this error didn't trigger any serious issue earlier.. Leftover of before I added a stride mechanism for custom attributes ;)

Reply:
It only comes into play with forwarded attributes, as buffers from `meshAlloc()` are contiguous (as returned by malloc/new). I guess the forwarding feature was too well-hidden before ;)

Reply:
Yeah "hidden" as in "not working" 😅